driver/src/main/scala/api/commands/GroupAggregation.scala

```diff
@@ -5,7 +5,7 @@ import reactivemongo.api.SerializationPack
 private[commands] trait GroupAggregation[P <: SerializationPack] {
   aggregation: AggregationFramework[P] =>
 
-  @inline private def document(name: String, arg: pack.Value): pack.Document =
+  @inline private def $document(name: String, arg: pack.Value): pack.Document =
```

Comment:
Do see the point

Reply:
In 2.10 I have 6 compilation errors like these:

```
[info] Compiling 33 Scala sources and 1 Java source to /home/joao/git/ReactiveMongo/driver/target/scala-2.10/test-classes ...
[error] /home/joao/git/ReactiveMongo/driver/src/test/scala/AggregationSpec.scala:105:19: method document in trait GroupAggregation cannot be accessed in c.AggregationFramework
[error]             Match(document("totalPop" -> document(f"$$gte" -> 10000000L))))
```

Because there is a clash between this `reactivemongo.api.commands.GroupAggregation#document` method and the `reactivemongo.bson.`package`#document` method even though the first is private...

This is not a problem in 2.11 or 2.12...

Reply:
ok, will try to do that later